test/AlgebraicGeometry/Schemes/FunctionFields.jl

```diff
@@ -1,3 +1,26 @@
+const rng = Oscar.get_seeded_rng()
+
+function test_elem(K::VarietyFunctionField)
+  F = representative_field(K)
+  P = base_ring(F)::MPolyRing
```

Comment:
Why the type assertion @HechtiDerLachs ?

Reply:
To assist the reader of the code. Nothing more.